data.json

```diff
@@ -1510,5 +1510,11 @@
     "githubUsername": "fabdul88",
     "imageUrl": "https://user-images.githubusercontent.com/60126985/138183204-9d6d4458-01e7-4394-816a-0d530a6b78ce.png",
     "issueNumber": 786
+  },
+  {
+    "name": "Eshan Gupta",
+    "githubUsername": "eshan1925",
+    "imageUrl": "https://user-images.githubusercontent.com/78701779/138263809-68e859b3-5cb5-4e87-b006-23499ebe66e9.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/138337875-218a672c-8400-4e1d-b7ef-e7ddfcb0f5e5.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/138337875-218a672c-8400-4e1d-b7ef-e7ddfcb0f5e5.png",
```